### PR TITLE
React Native support

### PR DESCRIPTION
**Pull request by Yamaguchi**
This PR add the feature for React Native.
Components implemented in this PR (ReactDataStore/ReactKeyStore) is based on Expo framework(https://docs.expo.io/)

### CODE REVIEW

**Comment by Yamaguchi** [comment]
Sample Application is here:

https://github.com/chaintope/tapyrusjs-react-example/

File: ts_src/index.ts [hunk 1]
```diff
@@ -18,4 +20,6 @@ export {
   FeeProvider,
   FixedFeeProvider,
   SizeBasedFeeProvider,
+  Balance,
+  Utxo,
```
Comment (rantan) [inline ts_src/index.ts:24]: I couldn't find balance.ts and utxo.ts. These are neccessary?

**Review by rantan — CHANGES_REQUESTED** [review]
How about key stores make commonize methods except get(), set(), remove().  Key store implementations seem that most methods are really similar in each file.

File: ts_src/index.ts (hunk 1 above)
Comment (Yamaguchi) [reply ts_src/index.ts:24]: https://github.com/chaintope/tapyrusjs-wallet/blob/main/ts_src/utxo.ts
https://github.com/chaintope/tapyrusjs-wallet/blob/main/ts_src/balance.ts

**Pull request merged**